common/src/main/proto/lobby/contracts/common/common.proto

```diff
@@ -21,10 +21,14 @@ syntax = "proto3";
 
 package lobby.contracts.common;
 
-option java_package="lobby.contracts.common";
+option java_package="sample.lobby.contracts.common";
```

Comment:
Let's have the following package for all samples: `org.spine3.samples.*`. 
And `lobby` would be one of them.